addon/services/prefetch.js

```diff
@@ -28,11 +28,20 @@ if (gte('3.6.0')) {
                 let { route, fullParams } = changeSet.for[i];
                 if (seenRoutes.has(route)) continue;
 
-                  route._prefetched = new RSVP.Promise(r => {
-                    return r(route.prefetch(fullParams, transition))
-                  });
-                  seenRoutes.set(route, true);
-                  if (transition.isAborted) return;
+                route._prefetched = new RSVP.Promise(r => {
```

Comment:
Actually I realize looking at this now that is does not really make sense to set `_prefetched` on the promise...